tests/integration-tests/tests/api/test_api.py

```diff
@@ -220,6 +237,10 @@ def _test_stop_compute_fleet(region, cluster_compute_fleet_client, cluster_insta
     compute_node_map = _test_describe_cluster_compute_nodes(region, cluster_instances_client, cluster_name)
     instances_to_terminate = _get_instances_to_terminate(compute_node_map)
 
+    # EC2 instances in "running" or "stopping" state cannot be terminated, so we will wait for the instances we
```

Comment:
comment is confusing, you are stating that instance in "running" cannot be terminated and then you wait instance to be running to terminate them. Can you please clarify?

Reply:
Typo, I meant to write "pending".